libcontainer/factory_linux.go

```diff
@@ -346,11 +347,25 @@ func (l *LinuxFactory) StartInitialization() (err error) {
 	envInitPipe := os.Getenv("_LIBCONTAINER_INITPIPE")
 	pipefd, err := strconv.Atoi(envInitPipe)
 	if err != nil {
+		logrus.Errorf("unable to convert _LIBCONTAINER_INITPIPE: %v", err)
 		return fmt.Errorf("unable to convert _LIBCONTAINER_INITPIPE: %w", err)
```

Comment:
```go
err = fmt.Errorf("unable to convert _LIBCONTAINER_INITPIPE: %w", err)
logrus.Error(err)
return err
```

(or something similar -- the idea is to not duplicate the error wrapping).

Reply:
Thanks for your feedback~ fixed.